code/modules/power/singularity/collector.dm

```diff
@@ -94,65 +104,62 @@
 		if(panel_open)
 			to_chat(user, "<span class='warning'>Close the maintenance panel first!</span>")
 			return TRUE
-		if(!user.transferItemToLoc(W, src))
+		if(!user.transferItemToLoc(item, src))
 			return
-		loaded_tank = W
+		loaded_tank = item
 		update_appearance()
-	else if(W.GetID())
-		if(allowed(user))
-			if(active)
-				locked = !locked
-				to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] the controls.</span>")
-			else
-				to_chat(user, "<span class='warning'>The controls can only be locked when \the [src] is active!</span>")
-		else
+	else if(item.GetID())
+		if(!allowed(user))
 			to_chat(user, "<span class='danger'>Access denied.</span>")
+		if(!active)
+			to_chat(user, "<span class='warning'>The controls can only be locked when \the [src] is active!</span>")
 			return TRUE
+		locked = !locked
+		to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] the controls.</span>")
+		return TRUE
 	else
 		return ..()
 
-/obj/machinery/power/rad_collector/wrench_act(mob/living/user, obj/item/I)
-	..()
-	default_unfasten_wrench(user, I)
+/obj/machinery/power/rad_collector/wrench_act(mob/living/user, obj/item/item)
+	. = ..()
```

Comment:
why set . if we don't use it? consistency?

Reply:
yes, we use . = ..() to call parent everywhere